microcloud/service/lxd_config.go

```diff
@@ -93,7 +96,7 @@ func (s LXDService) DefaultOVNNetwork(ipv4Gateway string, ipv4Range string, ipv6
 	}
 
 	ovnNetwork := api.NetworksPost{
-		NetworkPut: api.NetworkPut{Config: map[string]string{"network": "UPLINK"}, Description: "Default OVN network"},
+		NetworkPut: api.NetworkPut{Config: map[string]string{"network": DefaultOVNNetwork}, Description: "Default OVN network"},
 		Name:       "default",
```

Comment:
Given that we also have `"default"` as an `ovn` type network, I think the const name might be a bit misleading.

How about using `DefaultOVNNetwork` for `"default"` and `OVNUplinkNetwork` for `"UPLINK"`?

Reply:
I strongly agree. The uplink isn't an ovn network